Formula/openimageio.rb

```diff
@@ -0,0 +1,75 @@
+class Openimageio < Formula
+  desc "Library for reading, processing and writing images"
+  homepage "http://openimageio.org"
+  url "https://github.com/OpenImageIO/oiio/archive/Release-1.8.7.tar.gz"
+  sha256 "ed23ecdd3e4037a411c69a6a681273d01d4a2fbcc223eac86daba77dd03fc5ed"
+  head "https://github.com/OpenImageIO/oiio.git"
+
+  bottle do
+    cellar :any
+    sha256 "77bdd8b9b3e1b1ca6abd7eaa98227b0794b98b43fbacb24348774c4de10a4570" => :high_sierra
+    sha256 "b50f595ad597de8774ac929917d0371d2dca702349e3e4538250dc9fcf0050fd" => :sierra
+    sha256 "02adc761d2d21864e480795a4ef527de7852401770dec5dac31ea99872347001" => :el_capitan
+  end
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "boost"
+  depends_on "openssl"
+  depends_on "ffmpeg"
+  depends_on "freetype"
+  depends_on "giflib"
+  depends_on "ilmbase"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libtiff"
+  depends_on "opencolorio"
+  depends_on "openexr"
+  depends_on "openjpeg"
+  depends_on "webp"
+  depends_on "cfitsio" => :optional
+  depends_on "field3d" => :optional
+  depends_on "libraw" => :optional
+  depends_on "ptex" => :optional
+  depends_on "opencv@2" => :optional
+  depends_on "python3" => :optional
+
+  depends_on "boost-python" => (build.with?("python3") ? ["with-python3", "without-python"] : [])
```

Comment:
We'll just do python2 support for this formula.

Reply:
Why is this an audit violation? May I suggest to do python3 support instead?

Reply:
```
depends_on "foo" => "with-bar"
```
is an audit violation because it triggers a source build, and we only want to be pouring bottles for dependencies. The `(build.with? ...` syntax is defeating the audit but it still applies.

Since `--with-python3` is not the default for `boost-python`, we can only support python2 in this formula.

Reply:
Hmm, I see. Thank you for the explanation. Thing is, that using openimageio with python3 is my only use case. Without python3 support this formula is useless for me. Of course, I can only speak for myself.

Reply:
OK. If you create a `boost-python3` formula (and strip out the `--with-python3` option from `boost-python`), then we could depend on both of them, and support both python2 and python3.

Reply:
Same thing came to my mind.